INFO - 2025-02-07 19:38:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-02-07 19:38:18 --> CodeIgniter\HTTP\Exceptions\BadRequestException: The URI you submitted has disallowed characters: "]"
[Method: GET, Route: %5D]
in SYSTEMPATH\Router\Router.php on line 737.
 1 SYSTEMPATH\Router\Router.php(204): CodeIgniter\Router\Router->checkDisallowedChars(']')
 2 SYSTEMPATH\CodeIgniter.php(834): CodeIgniter\Router\Router->handle(']')
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2025-02-07 19:38:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-07 19:43:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-07 19:45:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-07 19:45:49 --> Error connecting to the database: mysqli_sql_exception: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\projetointegrador-teste\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\projetointegrador-teste\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'reale_gelato', 3306, '', 0)
#1 C:\xampp\htdocs\projetointegrador-teste\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\projetointegrador-teste\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projetointegrador-teste\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projetointegrador-teste\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projetointegrador-teste\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projetointegrador-teste\app\Controllers\Main.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projetointegrador-teste\system\CodeIgniter.php(933): App\Controllers\Main->index()
#8 C:\xampp\htdocs\projetointegrador-teste\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#9 C:\xampp\htdocs\projetointegrador-teste\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projetointegrador-teste\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\projetointegrador-teste\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\projetointegrador-teste\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente in C:\xampp\htdocs\projetointegrador-teste\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\projetointegrador-teste\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\projetointegrador-teste\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projetointegrador-teste\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\projetointegrador-teste\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\projetointegrador-teste\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\projetointegrador-teste\app\Controllers\Main.php(17): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\projetointegrador-teste\system\CodeIgniter.php(933): App\Controllers\Main->index()
#7 C:\xampp\htdocs\projetointegrador-teste\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#8 C:\xampp\htdocs\projetointegrador-teste\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\projetointegrador-teste\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\projetointegrador-teste\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\projetointegrador-teste\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-02-07 19:45:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `catalogos`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Main.php(17): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Main->index()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2025-02-07 19:46:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-02-07 19:46:10 --> mysqli_sql_exception: Table 'reale_gelato.catalogos' doesn't exist in C:\xampp\htdocs\projetointegrador-teste\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\projetointegrador-teste\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\projetointegrador-teste\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\projetointegrador-teste\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\projetointegrador-teste\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\projetointegrador-teste\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projetointegrador-teste\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\projetointegrador-teste\app\Controllers\Main.php(17): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\projetointegrador-teste\system\CodeIgniter.php(933): App\Controllers\Main->index()
#8 C:\xampp\htdocs\projetointegrador-teste\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
#9 C:\xampp\htdocs\projetointegrador-teste\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projetointegrador-teste\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\projetointegrador-teste\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\projetointegrador-teste\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-02-07 19:46:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'reale_gelato.catalogos' doesn't exist
[Method: GET, Route: /]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `catalogos`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Main.php(17): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Main->index()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-07 19:46:10 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'reale_gelato.catalogos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `catalogos`')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `catalogos`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `catalogos`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Main.php(17): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Main->index()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-02-07 19:46:10 --> [Caused by] mysqli_sql_exception: Table 'reale_gelato.catalogos' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *
FROM `catalogos`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `catalogos`')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `catalogos`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `catalogos`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Main.php(17): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Main->index()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Main))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2025-02-07 19:47:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-07 19:47:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-07 19:47:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-07 19:47:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-07 20:18:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-07 20:18:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-07 20:19:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-07 20:19:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-07 20:19:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-02-07 20:19:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
